server/rpc/interceptors/interceptors.go

```diff
@@ -409,9 +435,10 @@ func GetUnaryInterceptor(env environment.Env) grpc.ServerOption {
 		requestContextProtoUnaryServerInterceptor(),
 		authUnaryServerInterceptor(env),
 		quotaUnaryServerInterceptor(env),
+		copyHeadersUnaryServerInterceptor(),
+		identityUnaryServerInterceptor(env),
```

Comment:
ok don't laugh but have you run any perf tests to see the impact of this?

Reply:
under cacheload, the impact is about 0.5% CPU